psa_builder.py

```diff
@@ -204,16 +204,35 @@ def check_and_clone_repo(name, deps, dir):
     if not isdir(join(dir, name)):
         logging.info("Cloning %s repo", name)
         cmd = ["git", "-C", dir, "clone", "-b", gitref, deps.get(name)[0]]
-        run_cmd_and_return(cmd)
-        logging.info("Cloned %s repo successfully", name)
+        ret = run_cmd_and_return(cmd)
+        if ret == 0:
+            logging.info("Cloned %s repo successfully", name)
+        else:
+            logging.critical("Failed to clone %s repo, error: %d", name, ret)
+            sys.exit(1)
     else:
         logging.info("%s repo exists, fetching latest...", name)
         cmd = ["git", "-C", join(dir, name), "fetch"]
-        run_cmd_and_return(cmd)
-        logging.info("%s repo exists, checking out %s...", name, gitref)
+        ret = run_cmd_and_return(cmd)
+        if ret != 0:
+            logging.critical(
+                "Failed to fetch the latest %s, error: %d", name, ret
+            )
+            sys.exit(1)
+        logging.info("Checking out %s...", gitref)
```

Comment:
``` suggestion
        if ret != 0:
            logging.critical(
                "Failed to fetch the latest %s, error: %d", name, ret
            )
            sys.exit(1)

        logging.info("Checking out %s...", gitref)

Reply:
Done. Added an empty line after the `if` block.